src/assets/css_properties

```diff
@@ -1,3 +1,12 @@
+-epub-caption-side
+-epub-text-combine
+-epub-text-emphasis
+-epub-text-emphasis-color
+-epub-text-emphasis-style
+-epub-text-orientation
+-epub-text-transform
+-epub-word-break
+-epub-writing-mode
```

Comment:
Is this valid in Mint?

Reply:
It is, and properties starting with `-` (mostly browser prefixed) are not checked so we don't need to add these.

Reply:
Could we also check those? What other properties are there? Besides `-webkit-`, `-moz-`, `-ms-`, `-o-`?

Reply:
https://github.com/mint-lang/mint/blob/master/src/type_checkers/css_definition.cr#L21

Reply:
We could check those but there are a lot of them and keeping those up to date would be a challenge.

Also they are not used very much so I don't think it worth the effort.

Reply:
Looking at Autoprefixer, the mentioned prefixes are the only ones they use.
But likes you said, now days it is automated.

Reply:
Looks good to merge!